### DISCUSSION TITLE
Change innodb_ft_min_token_size

### DISCUSSION THREAD

**Discussion by netuser2004** [planetscale/discussion#413] [FAQs] [answered]
Is it possible to change the **innodb_ft_min_token_size** of my PlanetScale database from 3 (the default) to 1?  I've tried changing it by running the command **set global innodb_ft_min_token_size = 1;** but even though I get a response saying it was successful, running the command: **show global variables like 'innodb_ft_min_token_size';** to check it, shows it has not changed.  Can anyone help?

**Comment by fooforge** [comment, marked as answer]
​Hi @netuser2004,
​
Global variables such as `innodb-ft-min-token-size` cannot be changed on our [free, Scaler, and Team plans](https://planetscale.com/pricing) yet, I'm afraid. The only variables that can be changed during runtime are session variables. We have an internal feature request open where we're tracking demand and which I'm going to update shortly. I cannot say, however, if or when we're going to get to it at this time.
